cmd/migrate_from_qdrant.go

```diff
@@ -263,7 +260,7 @@ func (r *MigrateFromQdrantCmd) perpareTargetCollection(ctx context.Context, sour
 
 		if targetCollectionExists {
 			fmt.Print("\n")
-			pterm.Info.Printfln("Target collection already exists: %s. Skipping creation.", targetCollection)
+			pterm.Info.Printfln("Target collection '%s' already exists. Skipping creation.", targetCollection)
```

Comment:
are we following here any particular convention on user messages?

Reply:
No.

Reply:
Because `"Target collection 'collection-name' already exists. Skipping creation."` is more readable than `"Target collection already exists: collection-name. Skipping creation.`

Reply:
> Because `"Target collection 'collection-name' already exists. Skipping creation."` is more readable than `"Target collection already exists: collection-name. Skipping creation.`

It is a matter of taste.